### PR TITLE
upgrade: kured from 1.9.1 to 1.15.0

### PR DESCRIPTION
**Pull request by SeanDougherty**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Upgraded from 1.9.1 to 1.15.0

###### Change Log  

- Upgraded from 1.9.1 to 1.15.0

###### Does this affect the toolchain?  

NO

###### Test Methodology

- local build
- Buddy Build - [Link](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=499193&view=results)

### CODE REVIEW

File: SPECS/kured/kured.spec [hunk 1]
```diff
@@ -24,8 +24,8 @@
 %global debug_package %{nil}
 Summary:        Kubernetes daemonset to perform safe automatic node reboots
 Name:           kured
-Version:        1.9.1
-Release:        15%{?dist}
+Version:        1.15.0
```
Comment (mfrw) [inline SPECS/kured/kured.spec:27]: Can you please fill the PR checklist and also fill the testing method, local build/pipeline build.

Comment (SeanDougherty) [reply SPECS/kured/kured.spec:27]: Updated, here is the buddy build! [[Link](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=499195&view=results)]

Comment (SeanDougherty) [reply SPECS/kured/kured.spec:27]: > Can you please fill the PR checklist and also fill the testing method, local build/pipeline build.

Thank you @mfrw , your feedback has been incorporated! Could I get your follow-up review when you have the chance? Thanks!

**Review by mfrw — APPROVED** [review]
LGTM :)

**Comment by SeanDougherty** [comment]
TODO: Update patch format from `patch#` to `patch #`

**Comment by SeanDougherty** [comment]
> TODO: Update patch format from `patch#` to `patch #`

resolved

**Pull request merged**